packages/cli/src/databases/entities/test-run.ee.ts

```diff
@@ -69,8 +70,14 @@ export class TestRun extends WithTimestampsAndStringId {
 	 * Optional details about the error that happened during the test run
 	 */
 	@Column(jsonColumnType, { nullable: true })
-	errorDetails: IDataObject;
+	errorDetails: IDataObject | null;
 
 	@OneToMany('TestCaseExecution', 'testRun')
 	testCaseExecutions: TestCaseExecution[];
+
+	/**
+	 * Calculated property to determine the final result of the test run
+	 * depending on the statuses of test case executions
+	 */
+	finalResult?: TestRunFinalResult | null;
```

Comment:
Not sure how I feel about adding properties that are unrelated to what goes into the DB to entities 🤔 I guess it's fine